src/include/sof/math/log_10.h

```diff
@@ -0,0 +1,20 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2021 Intel Corporation. All rights reserved.
+ *
+ * Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+ *
+ */
+
+#ifndef BASEE_10_LOG_H
+#define BASEE_10_LOG_H
+
+/* Include Files */
+#include "stdint.h"
+#include <stddef.h>
+#include <stdlib.h>
+#define BASE_10_COMMON_LOGARITM 0xA0000000
```

Comment:
typo: LOGARITHM
and I would recommend to name as something like NUM_10_UQ4P28
(Just found that the Q-format expression of unsigned number could be UQm.n: https://en.wikipedia.org/wiki/Q_(number_format))